src/gui/ocwizard_deprecated.h

```diff
@@ -0,0 +1,25 @@
+#include <QObject>
+
+/*
+ * These functions are all deprecated and were previously located within the wizard class.
+ */
+
+namespace OCC::OwncloudWizard {
+
+[[deprecated]] static inline void askExperimentalVirtualFilesFeature(QObject *, std::function<void(bool)>) { }
+
+[[deprecated]] static inline bool useVirtualFileSync()
+{
+    return false;
```

Comment:
Please also add runtime warnings in order to minimize the chance that we ship it in this state.